### ISSUE TITLE
Project 0 review

### ISSUE THREAD

**Issue by rishikaharithaMSIT** [akankshadhanewar/sw-dev#1]
Your website looks very nice. The colors and feel of the site is good. However, you have not used @media, bootstrap grid and scss.
The screen cast link is also not submitted.

**Comment by akankshadhanewar** [comment]
Yes akka I had some doubts on scss so, I don't used. Now I'm clear. I don't know that I should use grid in my webpage.
